src/Angor/Shared/Services/RelayService.cs

```diff
@@ -16,8 +16,10 @@ public class RelayService : IRelayService
         private readonly INetworkService _networkService;
         private readonly IRelaySubscriptionsHandling _subscriptionsHandling;
         private readonly ISerializer _serializer;
-        
-        
+
+        // https://github.com/block-core/nips/blob/peer-to-peer-decentralized-funding/3030.md
+        public const NostrKind Nip3030NostrKind = (NostrKind)3030;
```

Comment:
ProjectInfoEventKind?

Reply:
Once We have the NuGet package for our fork of Nostr-client we will add that enum to it

Reply:
Yes I agree